docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,134 @@
+In Cosmos Hub, the `MaxAgeNumBlocks` is set to 1000000 (i.e., ~70 days if we assume we need ~6 sec per block) and `MaxAgeDuration` is set to 172800000000000 ns (i.e., 2 days). Because of this check, we can easily exclude old evidence.
+
+###  Slashing for equivocation on the consumer
+In the single-chain case, slashing requires both the `infractionHeight` and the voting `power`.
+In order to slash on the provider for an equivocation on a consumer, we need to have both the provider's `infractionHeight` and voting `power`.
+Note that the `infractionHeight` on the consumer chain must be mapped to a height on the provider chain.
+Unless we have a way to find the corresponding `infractionHeight` and `power` on the provider chain, we cannot slash for equivocation on the consumer in the same way as we would slash in the single-chain case.
+
+
+The challenge of figuring out the corresponding `infractionHeight` and `power` values on the provider chain is due to the following trust assumption:
+
+- We trust the consensus layer and validator set of the consumer chains, _but we do not trust the application layer_.
+
+As a result, we cannot trust anything that stems from the _application state_ of a consumer chain.
+
+Note that when a relayer or a user sends evidence through a [MsgSubmitConsumerDoubleVoting](https://github.com/cosmos/interchain-security/pull/1232) message, the provider gets access to [DuplicateVoteEvidence](https://github.com/cometbft/cometbft/blob/v0.37.0/types/evidence.go#L35):
+```protobuf
+type DuplicateVoteEvidence struct {
+	VoteA *Vote `json:"vote_a"`
+	VoteB *Vote `json:"vote_b"`
+
+	// abci specific information
+	TotalVotingPower int64
+	ValidatorPower   int64
+	Timestamp        time.Time
+}
+```
+The "abci specific information" fields cannot be trusted because they are not signed. Therefore,
+we can use neither `ValidatorPower` for slashing on the provider chain, nor the `Timestamp` to check the evidence age. We can get the `infractionHeight` from the votes, but this `infractionHeight` corresponds to the infraction height on the consumer and **not** on the provider chain.
+
+
+## Proposed solution
+As a first iteration, we propose the following approach. At the moment the provider receives evidence of equivocation on a consumer:
+1. slash all the undelegations and redelegations using `slashFactor`;
+2. slash all delegations using as voting `power` the sum of the voting power of the misbehaving validator and the power of all the ongoing undelegations and redelegations.
+
+**Evidence expiration:** Additionally, because we cannot infer the actual time of the evidence (i.e., the timestamp of the evidence cannot be trusted), we do not consider _evidence expiration_ and hence old evidence is never ignored (e.g., the provider would act on 3 year-old evidence of equivocation on a consumer).
```

Comment:
I would add something in here about how we will not slash validators if they have reassigned their consumer key more than one unbonding period ago

Reply:
Added a paragraph on this.